lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -16,6 +16,13 @@ class Base
         POINTS_PER_MINUTE = 10_000 # Points represent engineering time to resolve issue
         BASE_POINTS = 30 * POINTS_PER_MINUTE
 
+        SEVERITIES = [
+          MAJOR = "major".freeze,
+          MINOR = "minor".freeze,
+        ].freeze
+
+        SEVERITY_THRESHOLD = 120 * POINTS_PER_MINUTE # > 2 hours of work is major
```

Comment:
FWIW: this variable should probably be called `MAJOR_SEVERITY_THRESHOLD`. Also, as you might expect, I find the comment unnecessary ;)